### DISCUSSION TITLE
Download primary images only

### DISCUSSION THREAD

**Discussion by sison54** [photoprism/photoprism#1730] [Q&A] [answered]
Is there an easy way to download only the primary photos when selecting a set? Currently when I do this I get all of the images in the images that are in the stack (live photos, edits, etc.).

**Comment by graciousgrey** [comment, marked as answer]
At the moment you can download single files from a stack using the files tab (https://docs.photoprism.org/user-guide/organize/edit/#edit-files). 

We plan to offer an option to decide what files to download and set a default: https://github.com/photoprism/photoprism/issues/449

**Reply by lastzero** [reply]
In fact, you may be able to change this in settings.yml, there's just no UI yet.

**Reply by sison54** [reply]
Awesome, thanks for pointing me to the already existing issue! I am not sure how I missed it. I will look closer at it and see if I can get it to work using the settings file.
